Mathlib/Analysis/Normed/Ring/WithAbs.lean

```diff
@@ -141,18 +141,18 @@ open WithAbs
 variable {K : Type*} [Field K] (v : AbsoluteValue K ℝ)
 
 /-- The completion of a field with respect to a real absolute value. -/
-abbrev completion := UniformSpace.Completion (WithAbs v)
+abbrev Completion := UniformSpace.Completion (WithAbs v)
```

Comment:
Strictly we should add a `deprecated alias` here for the `completion` spelling. No need to do it for all the other lemmas.